Code/MinimalLib/jswrapper.cpp

```diff
@@ -372,6 +372,15 @@ emscripten::val get_avalon_fp_as_uint8array(const JSMol &self) {
 }
 #endif
 
+#ifdef RDK_BUILD_MINIMAL_LIB_FRAGMENTATION
+emscripten::val get_matched_fragments(const JSMol &mol, unsigned int minCuts, unsigned int maxCuts, unsigned int maxCutBonds) {
+  auto obj = emscripten::val::object();
+  std::pair<std::vector<std::string>, std::vector<std::string>> pairs = fragmentMol(mol, minCuts, maxCuts, maxCutBonds);
+  obj.set("fragmentsFirst", pairs.first);
+  obj.set("fragmentsSecond", pairs.second);
+  return obj;
```

Comment:
There is a possibility to return native arrays for Uint32Array for example with typed_memory_view emscripten function, but not for sting arrays (no specific value for allocated memory of one random string), so in JS it is possible to call array.get(index) which is recommnded here https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html.